src/simtools/simtel/simulator_light_emission.py

```diff
@@ -205,79 +343,201 @@ def _make_light_emission_script(self):
 
         telpos_file = self._write_telpos_file()
 
-        command = f"rm {self.output_directory}/"
-        command += f"{self.le_application[0]}_{self.le_application[1]}.simtel.gz\n"
-        command += str(self._simtel_path.joinpath("sim_telarray/LightEmission/"))
-        command += f"/{self.le_application[0]}"
+        app_name, app_mode = self.le_application
+        parts = []
+        # cleanup previous outputs
```

Comment:
Please change this to raise an exception. Applications should not remove existing files on disk (this might lead to severe data losses).